src/frontend/hooks/server/deviceInfo.ts

```diff
@@ -1,4 +1,7 @@
+import {DeviceInfo} from '@mapeo/schema';
```

Comment:
If we're only using this as a type, maybe change this to `import type`?

Either of these:

```suggestion
import type {DeviceInfo} from '@mapeo/schema';
```

```suggestion
import {type DeviceInfo} from '@mapeo/schema';
```

Reply:
addressed via https://github.com/digidem/comapeo-mobile/pull/473/commits/48890d891f0e42d358474ad155916d70c1f329c8